lib/statsd.js

```diff
@@ -44,6 +43,8 @@ const Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
   if (! this.protocol) {
     this.protocol = PROTOCOL.UDP;
   }
+  this.cacheDns = options.cacheDns === true;
+  this.dnsTtlMilliseconds = options.dnsTtlMilliseconds || 60000;
```

Comment:
Can you add 60000 as a const above?  I know, I know, very much not in place in here.  :) Trying to have new code start to do this however.

Reply:
Sure, I understand 😄 . Done, please check.